nat-lab/tests/uniffi/libtelio_remote.py

```diff
@@ -188,12 +188,16 @@ def main():
     container_ip = sys.argv[2]
     port = int(sys.argv[3])
 
-    daemon = Pyro5.server.Daemon(host=container_ip, port=port)
-    wrapper = LibtelioWrapper(daemon)
-    daemon.register(wrapper, objectId=object_name)
-
-    daemon.requestLoop()
-    daemon.close()
+    try:
+        daemon = Pyro5.server.Daemon(host=container_ip, port=port)
+        wrapper = LibtelioWrapper(daemon)
+        daemon.register(wrapper, objectId=object_name)
+
+        daemon.requestLoop()
+        daemon.close()
+    except Exception as e:  # pylint: disable=broad-exception-caught
+        print(f"libtelio_remote error: {e}")
```

Comment:
I don't think this print is going to show up anywhere. It might better to write it to `tcli.log`